online/src/main/scala/ai/chronon/online/TTLCache.scala

```diff
@@ -33,7 +39,9 @@ class TTLCache[I, O](f: I => O,
                      contextBuilder: I => Metrics.Context,
                      ttlMillis: Long = 2 * 60 * 60 * 1000, // 2 hours
                      nowFunc: () => Long = { () => System.currentTimeMillis() },
-                     refreshIntervalMillis: Long = 8 * 1000 // 8 seconds
+                     refreshIntervalMillis: Long = 8 * 1000, // 8 seconds
+                     // same as ttlMillis, so behavior is unchanged barring an override
+                     failureTTLMillis: Long = 2 * 60 * 60 * 1000 // 2 hours
```

Comment:
I think we should lower this to 30 seconds - I am no longer at airbnb, but I think it would benefit airbnb too.

Reply:
Hmm, I feel 30 seconds may be too frequent as default. Some of these TTLCache is used to cache metadata, which doesn't get updated frequently anyway. 

But I can see why for things like BatchIr cache, a more frequent failure refresh is desired.

Reply:
Thanks @nikhilsimha. Our incident was caused by a stale but valid metadata. To mitigate it, we would have to flush the TTL cache. This should be addressed with @yuli-han 's recent work that we will only fetch active configs. 

I am curious what is the `failureTTLMillis` from Stripe side? @yizzlez 

For failure cases, I agree that we should use a lower TTL.